rmc/utils/constraint.py

```diff
@@ -1305,44 +1319,534 @@ def create_context_with_oe(
+    transcript_ht = gene_model.ht()
+    all_transcripts = transcript_ht.aggregate(
+        hl.agg.collect_as_set(transcript_ht.transcript)
+    )
+    transcripts_no_rmc = qc_pass_transcripts.difference(rmc_transcripts)
+    transcripts_not_searched = all_transcripts.difference(qc_pass_transcripts)
+    ht = ht.select_globals()
+    return ht.annotate_globals(
+        transcripts_not_searched=transcripts_not_searched,
+        transcripts_no_rmc=transcripts_no_rmc,
+        outlier_transcripts=outlier_transcripts,
+    )
+
+
+def format_rmc_browser_ht(freeze: int, overwrite_temp: bool) -> None:
     """
     Reformat annotations in input HT for release.
 
     This reformatting is necessary to load data into the gnomAD browser.
 
     Desired schema:
-    ---------------------------------------
+    ----------------------------------------
+    Global fields:
+        'p_value': float64
+        'transcripts_not_searched': set<str>
+        'transcripts_no_rmc': set<str>
+        'outlier_transcripts': set<str>
+    ----------------------------------------
     Row fields:
-        'transcript_id': str
+        'transcript': str
         'regions': array<struct {
-            'start': int32
-            'stop': int32
-            'observed_missense': int32
-            'expected_missense': float64
-            'chisq': float64
+            start_coordinate: locus<GRCh37>,
+            stop_coordinate: locus<GRCh37>,
+            start_aa: str,
+            stop_aa: str,
+            obs: int64,
+            exp: float64,
+            oe: float64,
+            chisq: float64,
+            p: float64
         }>
-
     ----------------------------------------
-    Key: ['transcript_id']
+    Key: ['transcript']
     ----------------------------------------
 
-    :param ht: Input Table.
-    :return: Table with schema described above.
+    :param freeze: RMC data freeze number.
+    :param overwrite_temp: Whether to overwrite temporary data.
+        If False, will read existing temp data rather than overwriting.
+        If True, will overwrite temp data.
+    :return: None; writes Table with desired schema to resource path.
     """
+    ht = rmc_results.versions[freeze].ht()
+
+    # Annotate start and stop coordinates per region
     ht = ht.annotate(
-        regions=hl.struct(
-            start=ht.section_start,
-            stop=ht.section_end,
-            observed_missense=ht.section_obs,
-            expected_missense=ht.section_exp,
+        start_coordinate=ht.interval.start,
+        stop_coordinate=ht.interval.end,
+    )
+
+    # Annotate amino acids
+    ht = annot_rmc_with_aa(ht, overwrite_temp)
+
+    # Remove missense O/E cap of 1
+    # (Missense O/E capped for RMC search, but
+    # it shouldn't be capped when displayed in the browser)
+    ht = ht.annotate(section_oe=ht.obs / ht.exp)
+
+    # Convert chi square to p-value
+    ht = ht.annotate(section_p_value=hl.dchisq(ht.section_chisq, 1))
```

Comment:
`dchisq` is calculating the probability density at that specific chisq value in the distribution, we want `pchisqtail` for the cumulative probability under the distribution at this chi square value or greater.
```suggestion
    ht = ht.annotate(section_p_value=hl.pchisqtail(ht.section_chisq, 1))
```